doc/lua_api.txt

```diff
@@ -6008,12 +6008,12 @@ object you are working with still exists.
 * `set_sprite(p, num_frames, framelength, select_horiz_by_yawpitch)`
     * Select sprite from spritesheet with optional animation and Dungeon Master
       style texture selection based on yaw relative to camera
+    * `select_horiz_by_yawpitch` is only possible with visual = `sprite`
     * `p`: {x=number, y=number}, the coordinate of the first frame
       (x: column, y: row), default: `{x=0, y=0}`
     * `num_frames`: number, default: `1`
     * `framelength`: number, default: `0.2`
-    * `select_horiz_by_yawpitch`: boolean, this was once used for the Dungeon
-      Master mob, default: `false`
+    * `select_horiz_by_yawpitch`: boolean, default: `false`
```

Comment:
Few line above:
>    * Select sprite from spritesheet with optional animation and Dungeon Master
      style texture selection based on yaw relative to camera

This newly added note should go into the overall function description.

Reply:
> * Select sprite from spritesheet with optional animation (visual = `sprite` or `upright_sprite`) and Dungeon Master
style texture selection based on yaw relative to camera (only with visual = `sprite`)

?

Reply:
Overall suggestion (forwarded from IRC) - while we're already at it, it might make sense to improve the documentation further.
```
* `set_sprite(p, num_frames, framelength, select_x_by_camera)`
    * Specifies and starts a sprite animation
    * Animations iterate along the frame `y` position.
    * `p`: {x=column number, y=row number}, the coordinate of the first frame
      default: `{x=0, y=0}`
    * `num_frames`: Total frames in the texture, default: `1`
    * `framelength`: Time per animated frame in seconds, default: `0.2`
    * `select_x_by_camera`: Only for visual = `sprite`. Changes the frame `x`
      position according to the view direction. default: `false`.
```